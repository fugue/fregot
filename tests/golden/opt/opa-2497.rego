# https://github.com/open-policy-agent/opa/issues/2497
package bug

english = {
  "one": 1,
  "two": 2,
  "three": 3
}

should_be_1 = ret {
  textual = "one"
  numeric = 1
  ret = {total |
    english[k] = v
    k = textual
    total = sum([numeric | english[_] = numeric])
  }
}

test_should_be_1 {
  should_be_1 == {1}
}
