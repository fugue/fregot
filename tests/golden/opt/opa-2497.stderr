english = {
  "three": 3,
  "one": 1,
  "two": 2
}
should_be_1 = ret {
  textual = "one"
  numeric = 1
  ret = {total |
    bug.english.k = v
    k = textual
    total = sum([numeric |
      bug.english._ = numeric
    ])
  }
}
test_should_be_1 {
  bug.should_be_1 = {1}
}
